lib/trento/application/usecases/clusters/clusters.ex

```diff
@@ -29,16 +29,22 @@ defmodule Trento.Clusters do
 
   @spec request_checks_execution(String.t()) :: :ok | {:error, any}
   def request_checks_execution(cluster_id) do
-    ClusterReadModel
-    |> Repo.get(cluster_id)
+    query =
+      from(c in ClusterReadModel,
```

Comment:
a style nitpick, we do not use parenthesis here (locals w/o parens) if you remove them it should still work